app/app.go

```diff
@@ -0,0 +1,70 @@
+package app
+
+import (
+	"database/sql"
+
+	"golang.org/x/crypto/bcrypt"
+)
+
+//nema pojma za db i web
+
+type Application struct {
+	UserService *UserService
+}
+type User struct {
```

Comment:
I would create a new file called `users.go` in the `app` package and move all of the user related logic from this file to that one. Image adding new logic and having to keep it in this single file.

In the future, we want to keep related logic in a localized manner, for example using files to bundle user logic or using a `users.go` file in the server package to add all the handlers related to user APIs there, etc. That's the reason why we have the `postgres/users.go` file, where we would implement user related queries for the `UserRepository`.